fastlane/Fastfile

```diff
@@ -1083,7 +1083,7 @@ platform :ios do
       'promo_screenshot_5.txt'
     ].each do |filename|
       source = File.join(Dir.pwd, 'appstoreres', 'metadata', 'source', filename)
-      destination = File.join(en_us_path, filename)
+      destination = File.join(en_us_path, filename.gsub('promo_', 'app_store_'))
```

Comment:
We rename the English screenshot title to `app_store_screenshot_%` like the other locales.